app/components/primer/task_list.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Primer
+  # Add a general description of component here
+  # Add additional usage considerations or best practices that may aid the user to use the component correctly.
+  # @accessibility Add any accessibility considerations
+  class TaskList < Primer::Component
```

Comment:
```suggestion
  class TaskLists < Primer::Component
```

Reply:
Should we consider writing a `TaskList` Component that uses `task-lists` under the hood? It would be conceptually simpler:

```erb
<%= render(Primer::TaskList.new) do |component %>
  <% component.item ... %>
```

Instead of:

```erb 
<%= render(Primer::TaskLists.new) do |component %>
  <% component.list do |list| %>
    <% list.item ... %>
```

Reply:
I think the intention is that `TaskLists` is a list of lists and each of those inner lists has items:

```
<task-lists sortable>
  <ul class="contains-task-list">
    <li class="task-list-item">
      <input type="checkbox" class="task-list-item-checkbox">
      Hubot
    </li>
    <li class="task-list-item">
      <input type="checkbox" class="task-list-item-checkbox">
      Bender
    </li>
  </ul>

  <ul class="contains-task-list">
    <li class="task-list-item">
      <input type="checkbox" class="task-list-item-checkbox">
      BB-8
    </li>
    <li class="task-list-item">
      <input type="checkbox" class="task-list-item-checkbox">
      WALL-E
    </li>
  </ul>
</task-lists>
```